CMakeLists.txt

```diff
@@ -125,6 +125,23 @@ set(SCAP_WORKBENCH_LINK_LIBRARIES
     Qt5::Widgets Qt5::XmlPatterns
     ${OPENSCAP_LIBRARIES})
 
+# ---------- RPATHS for linking
+
+# see https://gitlab.kitware.com/cmake/community/wikis/doc/cmake/RPATH-handling
+
+# when building, use the install RPATH
+# (but later on when installing)
+set(CMAKE_BUILD_WITH_INSTALL_RPATH ON)
```

Comment:
I'm no sure we need to set `RPATH` on Linux and Windows.
I don't see any harm, but no benefit either.

Reply:
I realized this is needed on Apple because we don't run `install` target, we just build and run a script to generate the `dmg`.
I would add a comment and make this Apple only, and let CMake re-link on `install` target on Linux.

Reply:
But even then, at build time, openscap libraries are in system path, https://travis-ci.org/OpenSCAP/scap-workbench/jobs/481474332#L2433.
And the `dmg` is generated with openscap libraries at `./scap-workbench.app/Contents/Frameworks/`